frontend/src/tests/users.test.tsx

```diff
@@ -0,0 +1,172 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react'
```

Comment:
:sunglasses: - I think these are the kinds of tests we discussed shooting for as far as just confirming the FE tries to make the requests we'd expect :tada: .